### PR TITLE
RCD in Autolathe

### LINKED ISSUE #2893: RCD ammo is not under any autolathe disk's recipe list.
I don't mind the RCD being one of a kind but I need the ammo..

### PR DESCRIPTION
**Pull request by EvenInDeathIStillServe**
Adds RCD to the advanced tools disk. Replaced the silver cost with 3 platinum.
Adds RCD ammo to the basic tool disk. 30 metal + 10 glass.
Fixes a runtime error when using the RCD on stuff that isn't a floor when not in deconstruction mode.
Fixes a spelling mistake in the circuit disk. Technomacer > TechnomaNcer

Fixes #2893

### CODE REVIEW

**Comment by SpadesSlick** [comment]
Do we even have a ways to get platinum in our codebase at this moment?

**Comment by NanakoAC** [comment]
wew
okay i thought about this a bit
Make the Ammo require Metal, Plastic and Silver.
Small quantities of silver can reliably be gotten by cannibalizing power cells

The RCD itself, make it require plasteel, plasma, and uranium. Things which need a mining trip to obtain

**Comment by EvenInDeathIStillServe** [comment]
Done. Please review the numbers I set, in case they might be unbalanced.

File: code/game/objects/items/weapons/RCD.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	throw_range = 5
 	w_class = ITEM_SIZE_NORMAL
 	origin_tech = list(TECH_ENGINEERING = 4, TECH_MATERIAL = 2)
-	matter = list(MATERIAL_STEEL = 15, MATERIAL_PLASTIC = 5, MATERIAL_SILVER = 3)
+	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_PLASMA = 10, MATERIAL_URANIUM = 10)
```
Comment (NanakoAC) [inline code/game/objects/items/weapons/RCD.dm]: these quantities are a bit high, cut them in half

**Comment by EvenInDeathIStillServe** [comment]
Changed the costs for the cyborg & mech RCD to be similar to the handheld one's.
Made it so that the mechfab accepts any material. The list starts with metal, glass and plastic, and the list gets expanded when you insert new materials.

**Pull request merged**